magento/pwa-studio
# [PWA-832] Localize Category

## Description

Localize all strings within the Category area of the application. Include localized strings in default en_US language pack

## Related Issue





https://jira.corp.magento.com/browse/PWA-832

## Acceptance 



### Verification Stakeholders

@sirugh
@dpatil-magento

### Specification



### Verification Steps


1. Go to category page.
2. User should see translated text based on store view selected.
3. Verify nested categories and navigation
4. Verify Categories without products.
5. Verify on all supported browser and devices.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
